Baikal/CMakeLists.txt

```diff
@@ -162,8 +162,9 @@ if (BAIKAL_EMBED_KERNELS)
     target_compile_definitions(Baikal PRIVATE BAIKAL_EMBED_KERNELS)
     add_custom_command(
         OUTPUT ${KERNEL_HEADER}
-        COMMAND ${PYTHON_EXECUTABLE} ${STRINGIFY_SCRIPT} ${Baikal_SOURCE_DIR}/Baikal/Kernels/CL/ .cl opencl > ${KERNEL_HEADER}
+        COMMAND ${PYTHON_EXECUTABLE} ${STRINGIFY_SCRIPT} ../Baikal/Kernels/CL/ .cl opencl > ${KERNEL_HEADER}
```

Comment:
Because opencl files contain includes in form "../Baikal/Kernels/CL/<...>.cl" and it's more convenient to pass and work with single string than parse "${Baikal_SOURCE_DIR}/Baikal/Kernels/CL/" to different relative paths.